addons/web/static/src/js/views/calendar/calendar_renderer.js

```diff
@@ -412,88 +520,19 @@ return AbstractRenderer.extend({
                 self.trigger_up('updateRecord', event);
             },
             eventClick: function (eventClickInfo) {
-                eventClickInfo.jsEvent.preventDefault();
-                eventClickInfo.jsEvent.stopPropagation();
-                var eventData = eventClickInfo.event;
-                self._unselectEvent();
-                $(self.calendarElement).find(self._computeEventSelector(eventClickInfo)).addClass('o_cw_custom_highlight');
-                self._renderEventPopover(eventData, $(eventClickInfo.el));
+                self.eventClick(eventClickInfo);
```

Comment:
Isn't the point of this method to enable you to override "_getFullCalendarOptions" and define a different behavior for the "eventClick" method?

So instead of redefining "eventClick" in your "WebsiteCalendarRenderer" on the enterprise side, you would instead override "_getFullCalendarOptions" and give your custom definition of eventClick within the returned values.

Kind of like it's done in the "web_enterprise" module.

Same goes for the other methods here below.

Can you double check?